jenkins-pipelines/perf-regression-throughput-30min-i4i.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    params: params,
+
+    backend: "aws",
+    aws_region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: "test-cases/performance/perf-regression.100threads.30M-keys-i4i.yaml",
+    sub_tests: ["test_write", "test_read", "test_mixed"],
+
+    timeout: [time: 350, unit: "MINUTES"]
```

Comment:
It is not needed. Let's remove this 'timeout' parameter.

Reply:
since this is going straight to an old branch, we need to make sure if it's relevant or not on the target branch

Reply:
@vponomaryov , this branch doesn't have the `getJobTimeouts` in it... so it is required in here